build.gradle

```diff
@@ -10,3 +10,8 @@ buildscript {
 
 apply plugin: 'com.github.ben-manes.versions'
 apply plugin: 'android-reporting'
+
+task wrapper(type: Wrapper) {
```

Comment:
I don't feel like this is necessary, since `gradle wrapper` already exists for updating versions.